build.psm1

```diff
@@ -898,7 +898,7 @@ function Get-PesterTag {
                     }
                     $values = $vAst.FindAll({$args[0] -is "System.Management.Automation.Language.StringConstantExpressionAst"},$true).Value
                     $values | ForEach-Object {
-                        if (@('REQUIREADMINONWINDOWS', 'REQUIRESUDOONUNIX', 'SLOW') -contains $_) {
+                        if (@('REQUIREADMINONWINDOWS', 'REQUIRESUDOONUNIX', 'SLOW', 'REQUIRENOSUDOONUNIX') -contains $_) {
```

Comment:
I wonder why we add 'REQUIRENOSUDOONUNIX' if we haven't 'REQUIRENOADMINONWINSOWS'?

Reply:
On Windows, tests are always run elevated (remoting tests, update-help, and some others require elevation).  Tests that explicitly require non-admin (there's a few since I wrote some to check for Access Denied) uses runas to start a non-admin user.

On Linux, tests are usually run not under sudo.  There's just one test (currently) that explicitly tests the case where you need to be root, but running as not root you should get an error.  Only a few tests that require sudo.  We could change this design to always require running the tests under sudo, but that feels worse than requiring tests to run elevated on Windows.

It may be worth discussing in a separate issue so we can unblock the nightly runs with this PR.

Reply:
I see we run tests on Windows with `-Unelevated` by default and elevated if 'REQUIREADMINONWINDOWS' present - I'd expect the same for Unix - run test unelevated by default (that we have already) and elevated if 'REQUIRESUDOONUNIX' present - I don't see why we need another new tag.
It is non-blocking comment.

Reply:
I was thinking more of the individual engineer and not the nightly.  I think we could support `-Unelevated` on UNIX by creating a non-root user account and running the tests under that account.  Created https://github.com/PowerShell/PowerShell/issues/6246 to track.

Reply:
We'll need update docs https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#admin-privileges-in-tests

Reply:
RequiresNoSudoOnUnix is not needed.  Please don't needlessly added a tag.

Reply:
we really don't want to add a tag for this, the default behavior should be to not use sudo